addon/components/simple-typeahead/template.hbs

```diff
@@ -1,16 +1,25 @@
 <div class="input">
-  <input
+  {{input
     type='text'
-    value={{value}}
-    placeholder={{placeholder}}
-    oninput={{action 'input'}}
-    onkeydown={{action 'keydown'}}
-  />
+    spellcheck=false
+    value=value
+    placeholder=placeholder
+    input=(action 'input')
+    keyDown=(action 'keydown')
+  }}
 
-  <button title='clear input' class='clear' disabled={{valueIsEmpty}} {{action 'clear'}}>&#10005;</button>
+  {{#if showReset}}
+    <i title='clear input' class='clear' {{action 'clear'}}>
```

Comment:
OK, icon is customizable via the `clearButton` param, which accepts a string and will properly render stringified html/svg.  See [the example](https://github.com/jameslaneconkling/ember-simple-typeahead/tree/master/tests/dummy/app/components/custom-clear-button-example).

Does this accomplish what you're looking for?  If so, rebase on master and update this PR.